lib/parse.js

```diff
@@ -345,8 +345,22 @@ class Parser {
                     } else {
                         base = build.nthRoot(...args)
                     }
+                } else if (node.name === 'int') {
+                    switch (args.length) {
+                    case 2:
+                        base = build.apply('int', [args[0], args[1]])
+                        break;
+                    case 3:
+                        base = build.apply('int', [args[0], args[1], args[2]])
+                        break;
+                    case 4:
+                        base = build.apply('int', [args[0], args[1], args[2], args[3]])
+                        break;
+                    default:
+                        throw new Error("integral takes between 2 - 4 args")
+                    }
```

Comment:
This could be simplified to
```
if (args.length >= 2 && args.length <= 4) {
  base = build.apply('int', args)
} else {
  throw new Error("integral takes between 2 and 4 args")
}
```